src/pages/gateway/transforms.md

```diff
@@ -5,28 +5,31 @@ description: How to apply transforms to your mesh to customize your responses.
 
 # Transforms
 
-While [handlers] let you bring outside sources into API Mesh for Adobe Developer App Builder, [transforms] allow you to modify the schema in order to control the contents of your GraphQL requests and responses.
+While [handlers] let you bring outside sources into API Mesh for Adobe Developer App Builder, [transforms] allow you to modify the schema to control the contents of your GraphQL requests and responses.
 
 The API Mesh currently supports the following [transforms]:
 
 -  [Rename](#rename)
 -  [Prefix](#prefix)
--  [Filter](#filter-schema)
--  [Replace](#replace)
--  [Type Merge](#type-merge)
+-  [Filter Schema](#filter-schema)
+-  [Replace Field](#replace-field)
+-  [Type Merging](#type-merging)
 -  [Naming Convention](#naming-convention)
 -  [Hooks](#hooks)
 
-Additionally, these transforms are available but are not fully supported at this time:
+Additionally, these transforms are available but are not fully supported by API Mesh at this time:
 
 -  [Encapsulate]
 -  [Federation]
+-  [Hoist field](https://the-guild.dev/graphql/mesh/docs/transforms/hoist)
```

Comment:
I noticed that field hoisting did not work when we attempted it, so I'm glad you added it to this list. Can you clarify what is meant by "available but not supported?" Does it mean that these don't work at all, or that they require a different syntax?

By comparison, we were able to use the Encapsulate function as described.